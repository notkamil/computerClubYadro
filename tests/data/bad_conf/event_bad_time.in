3
08:00 20:00
10
24:60 1 client1