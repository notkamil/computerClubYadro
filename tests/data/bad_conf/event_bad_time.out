24:60 1 client1